addon/core/handler.ts

```diff
@@ -81,23 +91,123 @@ export default class TableHandler {
     throw new Error('NotImplemented');
   }
 
-  // @ts-ignore
-  applyFilters(column: Column, filters: Filter[]): Promise<any> {
-    throw new Error('NotImplemented');
+  /**
+   * Set filters on a given column.
+   *
+   * @param {Column} column - The column on which to apply filters
+   * @param {Filter[]} filters - The array of filters to apply to the column
+   * @returns {Promise<any>}
+   */
+  async applyFilters(column: Column, filters: Filter[]): Promise<any> {
+    const _filters = filters.reduce((acc, v) => {
+      const filterWithSameKey = acc.find((filter) => filter.key === v.key)
+
+      if (filterWithSameKey) {
+        filterWithSameKey.value = v.value;
+      } else {
+        acc.push(v);
+      }
+
+      return acc;
+    }, [...column.filters])
+
+    column.filters = _filters;
+
+    return this.tableManager.upsertColumns({ columns: this.columns }).then(({ columns }) => {
+      this.columns = columns;
+      this.currentPage = 1;
+      this.fetchRows();
+    });
   }
 
-  // @ts-ignore
-  applyOrder(column: Column, order: Order): Promise<any> {
-    throw new Error('NotImplemented');
+  /**
+   * Apply ordering to a column
+   *
+   * @param {Column} column — The column we want to order by
+   * @param {OrderDirection} direction — The direction we want to order the column in
+   * @returns {Promise<any>}
+   */
+  async applyOrder(column: Column, direction: OrderDirection): Promise<any> {
+    const orderedColumn = this.columns.find((column) => column.order);
```

Comment:
instead of all the time find the ordered column, should we not keep a pointer on it ?